content/blog/posts/tji-travels-under-the-pink-dome.md

```diff
@@ -0,0 +1,28 @@
+---
+title: TJI Travels Under the Pink Dome
+subtitle: ""
+date: 2021-05-10
+authors:
+  - Eva Ruth Moravec
+---
+Every other year, the pink granite building that sits in the heart of Austin buzzes with activity for six months as lawmakers gather to debate, consider, toss out and approve legislation that governs life in Texas. 
+
+This is one of said years, and said lawmakers are currently tumbleweeding towards the finish line at the end of May, when the clock runs out for bills to make it to Gov. Greg Abbott's desk for one last vote. For the first time, I testified *for* two bills – both proposals that would enhance transparency and accountability in criminal justice. While neither is likely to become law this session, I'm grateful for the opportunity and learned a new thing or two about this sausage-making process of ours that I thought I'd share. 
+
+It all starts  weeks before the start of the Texas legislative session, when Texas representatives and senators [start to file bills](https://www.texastribune.org/2020/11/09/texas-bill-filing-legislative-session/). Then in January, lawmakers, lobbyists and advocates and everyday Texans raise their voices. Ambitiously, legislators file thousands of bills – [this year, some 8,829 ](https://legiscan.com/TX)– and the race is on. In order for a bill to become state law, it must be heard in and voted out of committees in both chambers, and then in both chambers' floor sessions, and finally considered by the governor. 
+
+I testified for two House bills: [HB 1810](https://www.texastribune.org/2020/11/09/texas-bill-filing-legislative-session/), which addresses the format in which agencies release data, got voted out of the massive State Affairs committee and even the full House; and [HB 2901](https://capitol.texas.gov/BillLookup/History.aspx?LegSess=87R&Bill=HB2901), which would address law enforcement agencies that misreport deaths of people in their custody, stalled after a hearing in Homeland Security and Public Safety.  
```

Comment:
Is it possible to summarize your testimony? It would be interesting to know what kind of questions you've gotten and especially related to TJI's activities.